dominode-bootstrapper/dominode_bootstrapper/geonodeadmin.py

```diff
@@ -181,22 +183,35 @@ def from_geonode_manager(cls, geonode_manager: GeoNodeManager):
         )
 
     def list_workspaces(self):
+        self.client.headers = self.headers
+
```

Comment:
Don't update the request headers on the http client. This prevents reusing it for other connections that may not expect JSON input/output (like the geonode ones).

do this instead:

```python
response = self.client.post(url, headers=self.headers, ...) 
```